h2o-algos/src/main/java/hex/gam/GAM.java

```diff
@@ -494,6 +491,7 @@ private void copyGLMtoGAMModel(GAMModel model, GLMModel glmModel) {
       model._rank = glmModel._output.bestSubmodel().rank();
       model._ymu = new double[glmModel._ymu.length];
       System.arraycopy(glmModel._ymu, 0, model._ymu, 0, glmModel._ymu.length);
+      model._parms = copyGLMParams2GAMParams(glmModel._effective_parms);
```

Comment:
should we really change _parms here? I think the point was to leave _parms as is, so this should probably copy to _effective_parms, no?

Reply:
If I copy to effective params, it won't work because param.solver is set during GLM training. after training it is copied to effective_param.solver in hex.ModelBuilder.Driver#computeEffectiveParameters. when I create GAM, I have solver = AUTO and its not set then nowhere so after GAM is trained and I will get to hex.ModelBuilder.Driver#computeEffectiveParameters I will be copying param.solver = AUTO to effective_prams.solver and end up with AUTO in effective parameters.. maybe I can do `model._parms = copyGLMParams2GAMParams(glmModel._parms);` like this effective params wouldn't interact with params. Or you think GLM params shouldnt interact with GAM params at all here?